app/models/ability.rb

```diff
@@ -241,7 +241,7 @@ def offers_package_abilities
   def order_abilities
     can :create, Order
     can %i[index show transition], Order, created_by_id: @user_id
-    can %i[update destroy], Order, created_by_id: @user_id, state: %w[draft submitted processing awaiting_dispatch]
+    can %i[update destroy], Order, created_by_id: @user_id, state: %w[draft submitted processing awaiting_dispatch dispatching]
```

Comment:
@abulsayyad123 I think you've added this to the wrong line... should be line 247 below as the issue affects order managers not order creators.